FAQ.md

```diff
@@ -268,6 +293,19 @@ To solve this problem, use the [store delete](https://github.com/pingcap/pd/tree
 
 The RocksDB compresses the key.
 
+#### TiKV master and slave use the same compression algorithm, why the results are different?
+
+Currently, some files of TiKV master have a higher compression rate, which depends on the underlying data distribution and RocksDB implementation. It is normal that the data fluctuates occasionally. The underlying storage engine adjusts data as needed.
+
+#### What are the features of TiKV block cache?
+
+TiKV has the Column Falimies feature of RocksDB. By default, the KV data is eventually stored in the 3 CFs (default, write and lock) within RocksDB.
```

Comment:
TiKV has the Column Falimies feature of RocksDB. -> TiKV implements the Column Family feature of RocksDB.